### ISSUE TITLE
FailedValidation' [spec.tikv.storageVolumes[0].name: Required value: name must not be empty, spec.tikv.storageVolumes[0].mountPath: Required value: mountPath must not be empty

### ISSUE THREAD

**Issue by mahjonp** [pingcap/tidb-operator#3595]
## Bug Report

**What version of Kubernetes are you using?**

Client Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.5", GitCommit:"20c265fef0741dd71a66480e35bd69f18351daea", GitTreeState:"clean", BuildDate:"2019-10-15T19:16:51Z", GoVersion:"go1.12.10", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.2", GitCommit:"f5743093fd1c663cb0cbc89748f730662345d44d", GitTreeState:"clean", BuildDate:"2020-09-16T13:32:58Z", GoVersion:"go1.15", Compiler:"gc", Platform:"linux/amd64"}

**What version of TiDB Operator are you using?**

TiDB Operator Version: version.Info{GitVersion:"v1.1.7", GitCommit:"8b9978015babfb5e70f1f0b1e1ce43de7203eea4", GitTreeState:"clean", BuildDate:"2020-11-14T04:57:07Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/amd64"}

**What storage classes exist in the Kubernetes cluster and what are used for PD/TiKV pods?**

```
NAME                 PROVISIONER                RECLAIMPOLICY   VOLUMEBINDINGMODE   ALLOWVOLUMEEXPANSION   AGE
standard (default)   k8s.io/minikube-hostpath   Delete          Immediate           false                  29d
```

```
No resources found.
```

**What's the status of the TiDB cluster pods?**

```
No resources found.
```

**What did you do?**

```
# IT IS NOT SUITABLE FOR PRODUCTION USE.
# This YAML describes a basic TiDB cluster with minimum resource requirements,
# which should be able to run in any Kubernetes cluster with storage support.
apiVersion: pingcap.com/v1alpha1
kind: TidbCluster
metadata:
  name: basic
spec:
  version: v4.0.7
  timezone: UTC
  pvReclaimPolicy: Delete
  discovery: {}
  pd:
    baseImage: pingcap/pd
    replicas: 1
    # if storageClassName is not set, the default Storage Class of the Kubernetes cluster will be used
    # storageClassName: local-storage
    requests:
      storage: "1Gi"
    config: {}
  tikv:
    baseImage: pingcap/tikv
    replicas: 1
    # if storageClassName is not set, the default Storage Class of the Kubernetes cluster will be used
    # storageClassName: local-storage
    requests:
      storage: "1Gi"
    config:
      storage:
        # In basic examples, we set this to avoid using too much storage.
        reserve-space: "0MB"
      rocksdb:
        wal-dir: "/var/lib/wal"
      titan:
        dirname: "/var/lib/titan"
    storageVolumes:
      - name: wal
        storageSize: "2Gi"
        mountPath: "/var/lib/wal"
      - name: titan
        storageSize: "2Gi"
        mountPath: "/var/lib/titan"
  tidb:
    baseImage: pingcap/tidb
    replicas: 1
    service:
      type: ClusterIP
    config:
      log:
         file:
         name: /var/log/tidb
         level: "warn"
    storageVolumes:
      - name: log
        storageSize: "2Gi"
        mountPath: "/var/log/tidb"
```

**What did you expect to see?**

```
No error
```

**What did you see instead?**

```
Events:
  Type     Reason            Age                 From                     Message
  ----     ------            ----                ----                     -------
  Warning  FailedValidation  21s (x22 over 10m)  tidb-controller-manager  [spec.tikv.storageVolumes[0].name: Required value: name must not be empty, spec.tikv.storageVolumes[0].mountPath: Required value: mountPath must not be empty, spec.tikv.storageVolumes[1].name: Required value: name must not be empty, spec.tikv.storageVolumes[1].mountPath: Required value: mountPath must not be empty]
```

**Comment by july2993** [comment]
just notice already fixed by #3586

**Comment by july2993** [comment]
fix by #3586 
need wait for next version v1.1.8
